app/views/layouts/application.html.erb

```diff
@@ -12,7 +13,7 @@
       <ul class="list" id="index">
       <% if logged_in? %>
         <li><%= link_to 'home', categories_path %></li>
-        <li>Welcome, <%= current_user.username.capitalize %></li>
+        <li>Welcome <%= link_to 'User', user_path(current_user)%></li>
```

Comment:
I turned it into a welcome message as opposed to a link to the user homepage because there's another link in the next nav bar with the user homepage. @bpd4tw